src/librustc_typeck/diagnostics.rs

```diff
@@ -211,6 +211,53 @@ Reference:
 http://doc.rust-lang.org/reference.html#trait-objects
 "##,
 
+E0036: r##"
+This error occurred when you pass too many or not enough type parameters to a
+method. Bad example:
```

Comment:
“Bad example” suggests that this example itself is of unsatisfactory quality, while this example simply demonstrates how such an error might be caused. I suggest dropping “Bad” here.